event-store/src/aggregator.rs

```diff
@@ -72,9 +72,14 @@ use store_query::StoreQuery;
 ///         }
 ///     }
 /// }
-/// #
-/// # struct DummyQuery;
-/// # impl StoreQuery for DummyQuery {}
+///
+/// struct DummyQuery;
+/// impl StoreQuery for DummyQuery {
+///   fn unique_id(&self) -> String {
+///     // Please read docs about unique_id
```

Comment:
You should change this to a note about generating a unique identifier from the contents of the query and its args here; it's vitally important for the cache to work correctly.

Reply:
It does depend on the impl, yes, but the desired outcome is the same; a unique cache key for the given query/args, in whatever format they might be in. I think it's worth at least clarifying this comment a bit more.